velox/type/tests/TypeTest.cpp

```diff
@@ -39,6 +39,25 @@ TEST(Type, Array) {
   EXPECT_EQ(n, 1);
 }
 
+TEST(Type, FixedLenArray) {
+  auto arr0 = FIXED_SIZE_ARRAY(3, INTEGER());
+  EXPECT_EQ("FIXED_SIZE_ARRAY(3)<INTEGER>", arr0->toString());
+  EXPECT_EQ(arr0->size(), 1);
+  EXPECT_STREQ(arr0->kindName(), "FIXED_SIZE_ARRAY");
+  EXPECT_EQ(arr0->isPrimitiveType(), false);
+  EXPECT_STREQ(arr0->elementType()->kindName(), "INTEGER");
+  int32_t n = 0;
+  for (auto& i : *arr0) {
```

Comment:
it might be more legible to just assert using `arr0->size()`, and `arr0->childAt()`, something like:

```
 EXPECT_EQ("INTEGER", arr0->childAt(0)->toString());
 EXPECT_THROW(arr0->childAt(1), std::whatever_exception);
```